CMakeLists.txt

```diff
@@ -459,6 +462,12 @@ add_custom_command(TARGET foedag-bin POST_BUILD
                   COMMAND ${CMAKE_COMMAND} -E copy 
                   ${PROJECT_SOURCE_DIR}/etc/device.xml        
                           ${CMAKE_CURRENT_BINARY_DIR}/share/foedag/etc/)
+                          
+add_custom_command(TARGET foedag-bin POST_BUILD
+                  COMMAND ${CMAKE_COMMAND} -E make_directory ${CMAKE_CURRENT_BINARY_DIR}/share/foedag/etc/settings/
+                  COMMAND ${CMAKE_COMMAND} -E copy 
+                  ${PROJECT_SOURCE_DIR}/etc/settings/settings_test.json
```

Comment:
why not add to the resources?

Reply:
I don't really know what our directory structure is yet. This is where Alain suggested to put it when i was moving it away from the root folder where I had it when testing/prototyping.

Reply:
I mean some resource.qrc file to make settings_test.json as a part of the project

Reply:
Ah I understand, this is just a short term file to demo the concept. I envision in the end we will have 1 or more settings definition files that we can read at startup to build our settings. Some of those will be files we deliver with the product and could be stored in a resource file (if the user never needs to see them), some would be files that downstream clients want to add, and finally there are user copies of .json files that contain only their changes to add on top of the settings data. In the end we will probably be accessing these files from multiple locations.